app/Models/Foundation/Summit/Events/Presentations/Presentation.php

```diff
@@ -2402,4 +2418,77 @@ public function getSnapshot():array{
 
         return array_merge( $snapshot, parent::getSnapshot());
     }
+
+    /**
+     * @return string
+     * @throws \Exception
+     */
+    public function getReviewStatus(): string
+    {
+        if ($this->selection_plan == null) return self::ReviewStatusNoSubmitted;
+
+        $submission_begin_date = $this->selection_plan->getSubmissionBeginDate();
+        $submission_end_date = $this->selection_plan->getSubmissionEndDate();
+        $selection_begin_date = $this->selection_plan->getSelectionBeginDate();
+        $selection_end_date = $this->selection_plan->getSelectionEndDate();
+        $submission_lock_down_presentation_status_date = $this->selection_plan->getSubmissionLockDownPresentationStatusDate();
+
+        $now = new \DateTime('now', new \DateTimeZone('UTC'));
+
+        /**
+         * Published - Presentation is published
+         * Not Submitted - the submission period is open or closed, the submission is started, but not complete.
+         * Received  - the submission is complete and the submission period is open
+         * In Review - the submission is complete, submission period is closed, track chairs is open or the
+         *             submission_lock_down_presentation_status_date is greater than now.
+         * Rejected - the submission is complete, the track chairs is closed, submission is closed, and the presentation is
+         *            not in alternate or accepted on teams list.
+         * Accepted - the submission is complete, the track chair is closed, submission is closed, and the presentation is
+         *            in alternate or accepted on teams list.
+         **/
+
+        // submission ( CFP )
+        // check submission period
+        $submission_open = $now >= $submission_begin_date && $now <= $submission_end_date;
+
+        // selection ( track chairs )
+        // check selection period
+        $selection_open = $now >= $selection_begin_date && $now <= $selection_end_date;
+
+        $presentation_status = $this->getStatus();
+
+        if (!in_array($presentation_status, [self::ReviewStatusReceived, self::ReviewStatusAccepted])) {
```

Comment:
@romanetar please lets not change the logic , lets be sure to follow the same logic as https://github.com/fntechgit/call-for-presentations/blob/b1952301be6962feb89913448bfdd4543c527823/src/model/presentation.js#L84